### PR TITLE
Redo PR #1081

### PR DESCRIPTION
**Pull request by AmandaQuad**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3364,6 +3370,7 @@
 			),
 			'sysop' => array(
 				'moderator',
+				'tagmanager'
```
Comment (ImBoPhil) [inline LocalSettings.php]: You need to add a comma here

File: LocalSettings.php [hunk 2]
```diff
@@ -2755,6 +2755,7 @@
...
```
Comment (ImBoPhil) [inline LocalSettings.php]: missing comma

**Comment by LegoFan4000** [comment]
Revoking is incorrect, repealing is.

**Comment by AmandaQuad** [comment]
@LegoFan4000 I don't know how to do that

File: LocalSettings.php (hunk 1 above)
Comment (AmandaQuad) [inline LocalSettings.php]: Done

File: LocalSettings.php (hunk 2 above)
Comment (AmandaQuad) [inline LocalSettings.php]: Done

**Comment by LegoFan4000** [comment]
https://github.com/miraheze/mw-config/blob/master/LocalWiki.php#L105

**Comment by LegoFan4000** [comment]
Good, now you just need to remove what you did here

File: LocalSettings.php [hunk 3]
```diff
@@ -3395,6 +3402,11 @@
  				'read' => true,
 				'edit' => true,
  			),
+			'user' => array(
+				'applychangetags' => true,
+				'managechangetags' => true,
+				'changetags' => true,
+			),
```
Comment (LegoFan4000) [inline LocalSettings.php]: Remve these 5 lines please.

File: LocalSettings.php [hunk 4]
```diff
@@ -3395,6 +3402,7 @@
  				'read' => true,
 				'edit' => true,
  			),
+
```
Comment (LegoFan4000) [inline LocalSettings.php]: This empty space is not needed.

Comment (AmandaQuad) [inline LocalSettings.php]: Will it really harm anything? It's so annoying having to scroll through 3000+ lines of code every time. If it absolutely needs to be removed, I'll ask someone else to do it for me.

**Pull request merged**